appcenter-crashes/scripts/postinstall.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs');
+const path = require('path');
+
+const projectDirectory = path.resolve(`${__dirname}${path.sep}..${path.sep}..${path.sep}..`);
+const testDirectory = path.join(projectDirectory, 'test');
+const setupFileName = 'setupAppCenter.js';
+const packageJsonFile = path.join(`${projectDirectory}`, 'package.json');
+
+// Update project.json
+let packageJsonContent;
+try {
+    packageJsonContent = fs.readFileSync(packageJsonFile, 'utf8');
+} catch (e) {
+    console.log('Could not read package.json file');
+    return;
+}
+const projectJson = JSON.parse(packageJsonContent);
+if (Object.prototype.hasOwnProperty.call(projectJson, 'jest')) {
+    const setupFileNameValue = `.${path.sep}test${path.sep}${setupFileName}`;
+    if (projectJson.jest.setupFiles === undefined) {
+        projectJson.jest.setupFiles = [setupFileNameValue];
+    } else if (projectJson.jest.setupFiles.indexOf(setupFileNameValue) === -1) {
+        projectJson.jest.setupFiles.push(setupFileNameValue);
+    }
+    fs.writeFileSync(packageJsonFile, JSON.stringify(projectJson));
+}
+
+// Create setup mock file for Jest
+if (!fs.existsSync(testDirectory)) {
+    fs.mkdirSync(testDirectory);
+}
+
+fs.writeFileSync(`${testDirectory}/${setupFileName}`, `
+jest.mock('NativeModules', () => ({
```

Comment:
How does `jest.mock` handle the scenario where the user has already defined a mock for `NativeModules`? Does this override/conflict with that? Or simply merge them?